### PR TITLE
Fixing status handler for analysis route + misc other status fixes

### PR DESCRIPTION
**Pull request by dcelik**
This PR adds the following:
 * Adds status checking for analysis route
 * Fixes some status checking calls

### CODE REVIEW

File: app/handlers/testConfig.py [hunk 1]
```diff
@@ -30,39 +30,37 @@ class TestConfigHandler(BaseHandler):
```
Comment (segerphilip) [inline app/handlers/testConfig.py]: I have made similar changes, might be a merge conflict in this one. Looks good though 👍

**Review by segerphilip — APPROVED** [review]
Looks good, test and 🚢

File: app/handlers/analysis.py [hunk 2]
```diff
@@ -26,11 +27,23 @@ class AnalysisHandler(BaseHandler):
...
-    def post(self):
-        identifier = self.get_body_argument("identifier")
+    def prepare(self):
+        self.identifier = self.get_body_argument("identifier")
+        status_dict = StatusHelper.get_status(self.identifier)
+        if status_dict[Status.Type.OBJECT_TRACKING] == Status.Flag.IN_PROGRESS or status_dict[Status.Type.SAFETY_ANALYSIS] == Status.Flag.IN_PROGRESS:
+            status_code = 423
+            self.error_message = "Currently analyzing your video. Please wait."
+            raise tornado.web.HTTPError(status_code = status_code)
+        if status_dict[Status.Type.CONFIG_HOMOGRAPHY] == Status.Flag.COMPLETE:
```
Comment (runnersaw) [inline app/handlers/analysis.py]: `!=`?

File: app/handlers/objectTracking.py [hunk 3]
```diff
@@ -33,23 +33,23 @@ class ObjectTrackingHandler(BaseHandler):
...
-        identifier = self.get_body_argument("identifier")
-        if StatusHelper.get_status(identifier)[Status.Type.OBJECT_TRACKING] == Status.Flag.IN_PROGRESS:
+        self.identifier = self.get_body_argument("identifier")
+        status_dict = StatusHelper.get_status(self.identifier)
+        if status_dict[Status.Type.OBJECT_TRACKING] == Status.Flag.IN_PROGRESS:
             status_code = 423
             self.error_message = "Currently analyzing your video. Please wait."
             raise tornado.web.HTTPError(status_code = status_code)
-        if StatusHelper.get_status(identifier)[Status.Type.CONFIG_HOMOGRAPHY] == Status.Flag.COMPLETE:
+        if status_dict[Status.Type.CONFIG_HOMOGRAPHY] == Status.Flag.COMPLETE:
```
Comment (runnersaw) [inline app/handlers/objectTracking.py]: Denizzzz

**Review by runnersaw — COMMENTED** [review]
wow

File: app/handlers/analysis.py (hunk 2 above)
Comment (segerphilip) [reply app/handlers/analysis.py]: Oh, and I missed this too 😢

Comment (dcelik) [reply app/handlers/analysis.py]: :(

**Pull request merged**